articles/react-native-agora-livestreaming-app/index.md

```diff
@@ -0,0 +1,681 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/react-native-agora-livestreaming-app/
+title: React Native Livestream Application using Agora
+description: This tutorial will give readers a detailed guide on how they can build an app to create or join a live stream using Agora.
+author: mohan-raj
+date: 2020-11-19T00:00:00-10:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/react-native-agora-livestreaming-app/hero.jpg
+    alt: Python example image PyPi
+---
+Want to learn how to build a cool and simple Livestreaming app using React Native? In this tutorial, we will do so using [Agora](https://www.agora.io/en/about-us/), which is a video, voice, and live interactive streaming platform. It includes embedded voice and video chat, real-time recording, interactive live streaming, and real-time messaging.
+<!--more-->
+Keep reading to find out.
+
+### Goals
+By the end of this tutorial, you’ll understand
+
+- How to create/join a live stream using Agora.
+
+- How to share a room code for others to join the live stream.
+
+- How to add event listeners on the live stream to listen to various state changes.
+
+### Prerequisites
+This article will not cover tutorial aspects of how React/React Native works. If you don't know how to work with it, please refer to [some tutorials](https://reactnative.dev/docs/tutorial) before beginning with this project.
```

Comment:
Since we are targeting beginner/intermediates, they need to know if they will be able to follow along. So you can rephrase this to be something like: "The fundamentals of React and React Native will not be covered in this tutorial, however, no prior knowledge is required to follow along in building this app" or something similar

Reply:
The fundamentals of React and React Native will not be covered in this tutorial. If you don't know the fundamentals, please refer to [some tutorials](https://reactnative.dev/docs/tutorial) before beginning with this project.

This is how I phrased it.